app/javascript/retrospring/features/inbox/entry/delete.ts

```diff
@@ -23,11 +22,9 @@ export function deleteEntryHandler(event: Event): void {
     cancelButtonText: I18n.translate('voc.cancel'),
     closeOnConfirm: true
   }, (returnValue) => {
```

Comment:
- It's observed that we are only invoking this callback if the user confirms the deletion
- It's not invoked when the `cancel` button is pressed. i.e. [line#25](https://github.com/Retrospring/retrospring/pull/1457/files#diff-aaf0f7f045dcad2e26621fb995727e628d3225898748c172e2137e8d5b96aec2R25) is a dead-code and `returnValue` is always `true`
- However, I've still handled the issue in this area as well just in case if I'm missing anything.
- If I've a green signal, I believe it's better to just remove this value and [line#25](https://github.com/Retrospring/retrospring/pull/1457/files#diff-aaf0f7f045dcad2e26621fb995727e628d3225898748c172e2137e8d5b96aec2R25) from the code as well.
- Please suggest the path forward. Thanks.